### PR TITLE
Address subteam and implementation tracking references

### LINKED ISSUE #3: Subteam is undefined
The README refers to "subteam" but it's unclear what is considered to be a subteam.
I can see @kchristidis replaced several occurrences with maintainers so I don't know whether the remaining references are intentional or a mere oversight.

### PR DESCRIPTION
**Pull request by kchristidis**
Closes #3, #4, and #6.

Signed-off-by: Kostas Christidis <kostas@christidis.io>

### CODE REVIEW

File: README.md
Comment (kchristidis) [inline README.md]: I think we should reconsider the "all maintainers" policy and move to a 2/3rds rule, or a similar majority rule. Otherwise we run the risk of being blocked by maintainers who are not active.

Comment (kchristidis) [reply README.md]: See #6.

Comment (lehors) [reply README.md]: I agree, requiring sign off from all the maintainers is not practical. This is actually why I wasn't sure the reference to subteam was merely an oversight. I thought it might refer to a subset of the maintainers.

**Review by lehors — APPROVED** [review]
Other than the issue related to the "all maintainers rule" duly noted, this PR addresses issues #3 and #4.
Thanks.

File: README.md
Comment (C0rWin) [reply README.md]: Why not to identify components and components owners so they will provide consent by signing-off?

**Comment by jyellick** [comment]
Agreed that all maintainer's is not practical.  But, since there is no such thing as a 'subteam' (at least as Fabric exists today), getting their agreement seems even less practical.  To me, it makes sense to remove the unknown terminology, rather than invent a definition to try to make the paragraph make sense.  If we want to address the 'all maintaners' bit (which I think we do), then that can would be another PR.

**Comment by kchristidis** [comment]
@jyellick: Is this a response to @C0rWin's comment, or to this PR in general? (I'm guessing the former, in which case I agree, but it was posted as a comment on the entire PR so the intent is a bit lost on me.)

**Comment by jyellick** [comment]
A bit of both.  Primarily it was meant in support of merging this PR as is, since 'subteam' is an undefined and meaningless term.

I was also in a way responding to @C0rWin with respect to his suggestion to identify component owners, that if we want to go that route, we can do it in another PR, but I'd rather we construct the text to match our definitions, rather than create definitions to match the text.

**Comment by C0rWin** [comment]
@jyellick I have just pointed out suggestion, wasn't trying to suggest making changes into current PR. I agree we have to make more concrete formulation which could be a different PR.

File: README.md
Comment (denyeart) [reply README.md]: Many rfcs will span components. I'd prefer we go with a simple majority of maintainers here.

Comment (christo4ferris) [reply README.md]: Agree that we need a majority of maintainers agreeing. All is impractical and even 2/3s seems extreme.

Comment (kchristidis) [reply README.md]: This PR is not meant to address this issue. This PR removes the subteam reference.

I will push a PR that closes #6 with a "majority" comment.

**Review by christo4ferris — APPROVED** [review]
LGTM thanks all

**Pull request merged**
